src/PreferencesDialog.cpp

```diff
@@ -434,6 +455,19 @@ void PreferencesDialog::removeExtension()
         ui->listExtensions->takeItem(ui->listExtensions->currentIndex().row());
 }
 
+void PreferencesDialog::listUpBuiltinExtension()
```

Comment:
What does "List Up" mean?

Reply:
Oh, I googled it and it's one of the most misused English expression in Korea. What I meant was 'make a list'. This function is responsible for getting the list of extension files in the app bundle and adding them to a new list widget.

Reply:
How would you recommended renaming it? I'm thinking of `makeListForBuiltinExtension`

Reply:
Maybe `BuiltinExtensionList`?

Reply:
No worries.  How about `createBuiltinExtensionList`?

If that's a bit lousy too, then just pick whatever you're happy with.  I just hadn't come across "List Up" as a phrase before. :smile:

Reply:
Cool, I've renamed the function as you suggested.
and I learned that "List Up" is an English expression used only in Korea. 😄